mbf_msgs/srv/CheckPath.srv

```diff
@@ -12,11 +12,11 @@ uint8                      skip_poses        # skip this number of poses between
 bool                       path_cells_only   # check only cells directly traversed by the path, ignoring robot footprint
                                              # (if true, safety_dist is ignored)
 ---
-uint8                      FREE      =  0    # robot is completely in traversable space
```

Comment:
would this change the md5 sum of the message?

Reply:
looks like not, as comments are removed before serializing
http://wiki.ros.org/ROS/Technical%20Overview#Message_serialization_and_msg_MD5_sums